source/installation/deploy-minio-standalone.rst

```diff
@@ -1,63 +1,45 @@
-===============================
-Deploy MinIO in Standalone Mode
-===============================
+=====================================
+Deploy Single-Node Single-Drive MinIO
+=====================================
 
 .. default-domain:: minio
 
 .. contents:: Table of Contents
    :local:
    :depth: 1
 
-The procedures on this page cover deploying MinIO in 
-:guilabel:`Standalone Mode`. A standalone MinIO deployment consists of a single
-MinIO server process with a single drive or storage volume ("filesystem mode").
-The MinIO server provides an S3 access layer to the drive or volume and stores
-objects as-is without any :ref:`erasure coding <minio-erasure-coding>`.
+The procedures on this page cover deploying MinIO in a Single-Node Single-Drive (SNSD) configuration for early development and evaluation.
+This mode was previously called :guilabel:`Standalone Mode` or 'filesystem' mode.
 
-For extended development or production environments, *or* to access
-:ref:`advanced MinIO functionality <minio-installation-comparison>` deploy MinIO
+For extended development or production environments, deploy MinIO
 in :guilabel:`Distributed Mode`. See :ref:`deploy-minio-distributed` for more
 information.
 
 Pre-Existing Data
 -----------------
 
-When starting a new standalone MinIO server, the storage path may have existing data already grouped into directories.
-MinIO displays this data as buckets and objects in the MinIO deployment.
+Starting with :minio-release:`RELEASE.2022-06-02T02-11-04Z`, MinIO implements a zero-parity erasure coded backend for single-node single-drive deployments.
+This feature allows access to :ref:`erasure coding dependent features <minio-erasure-coding>` without the requirement of multiple drives.
+The specified data volume or folder *must* be empty *or* only contain files generated by a previous |SNSD| deployment.
 
-- Top-level folders at the starting path become MinIO buckets.
-- Files in folders within the starting path become objects within the MinIO buckets.
-- Files in the top-level starting path do not display in MinIO.
+Use an earlier version of MinIO to retain the legacy filesystem-access behavior.
 
-Consider a starting path with the following structure:
+.. admonition:: MinIO Avoids Root Disks
+   :class: important
 
-.. code-block:: 
-   
-   /data
-      file.txt
-      /foo
-         file2.txt
-      /bar
-         file3.txt
+   MinIO by default avoids using the root OS disk as part of startup and returns an error if the specifid path is part of the root disk.
```

Comment:
/sp/ "if the **specified** path"
I'd lead with "By default, MinIO avoids" but that's a personal preference, I think.

Reply:
Agreed with tweak